test/integration/api-schema/_utils.js

```diff
@@ -48,30 +25,23 @@ exports.errorValidator = thunky(function (cb) {
   loadSchema('error.json', cb)
 })
 
-function loadSchema (relativePath, cb) {
-  schemaDir(function (err, dir) {
-    if (err) {
-      return cb(err)
+function loadSchema (schemaFileName, cb) {
+  const schemaPath = join(schemaDir, schemaFileName)
+  fs.readFile(schemaPath, { encoding: 'utf8' }, function (readErr, content) {
```

Comment:
Just noting out loud that we've removed the `console.log('processing "%s"...', path)` debugging output.  I'm presuming this is intention 👍

Reply:
Yup that was the intent: 1. I don't expect this to fail anymore, so isn't interesting. 2. That "processing ..." output isn't valid TAP output.